gms/gossiper.hh

```diff
@@ -87,6 +87,8 @@ class feature_service;
 struct bind_messaging_port_tag {};
 using bind_messaging_port = bool_class<bind_messaging_port_tag>;
 
+using advertise_myself = bool_class<class advertise_myself_tag>;
```

Comment:
Please document what this flag means from the perspective of gossiper user who will set the flag.

Reply:
Done.